docs/AccessControl.md

```diff
@@ -113,6 +113,246 @@ This specifies that the file is expected to be transmitted from the caller to
+but the session ID (in `s` key) is either not specified or it does
+specify a valid session, however the authorization region for this
+session does not embrace the specified resource.
+
+
+### SRT_REJX_NOTFOUND
+
+The resource specified in the `r` key (with combination of `h` key)
+is not found at this time. This error should be only reported if the
+information about resource accessibility is allowed to be publicly
+visible, otherwise the application might rather report authorization
+errors.
+
+
+### SRT_REJX_BAD_MODE
+
+The mode specified in the `m` key in StreamID is not supported for this request.
+This is in case of having read-only or write-only resources, as well as the
+interactive (bidirectional) access is not predicted as a valid access to this
+resource.
+
+
+### SRT_REJX_UNACCEPTABLE
+
+The requested parameters specified in SocketID cannot be satisfied for the
+requested resource. Also when m=publish and the data format is not acceptable.
+This is a general error reporting the unsupported data format for the data
+in case when it appears to be wrong when sending, or a restriction on the
+data format (as specified in the details of the resource specification) cannot
+be provided when receiving.
+
+
+### SRT_REJX_CONFLICT
+
+The resource being accessed is already locked for modification. This error
+should only be reported for `m=publish` in case when the resource being
+accessed is read-only. Note that it doesn't mean that the resource is not
+modifiable under current authorization restrictions, but that it is currently
+disabled completely for modifications.
+
+
+### SRT_REJX_NOTSUP_MEDIA
+
+The media type is not supported by the application. The media type is
+specified in the `t` key, although the currently standard types are
+`stream`, `file` and `auth`, an application may extend this list, and
+also it's not obliged to support all of the standard ones.
+
+
+### SRT_REJX_LOCKED
+
+The resource being accessed is locked for any access. This is similar to
+`SRT_REJX_CONFLICT`, but in this case it's locked for reading and writing.
+This is for a case when the resource should be shown as existing and in
+general available for the client, just temporarily blocked for access at
+the moment.
+
+
+### SRT_REJX_FAILED_DEPEND 
+
+The dependent entity for the request is not present. In this case the
+dependent entity is the session, which should be specified in the `s`
+key. This means that the specified session ID is nonexistent or it
+has already expired.
+
+
+### SRT_REJX_ISE
+
+Internal server error. This is for a general case when the request has
+been correctly verified and no problems related to this have been found,
+the processing of the request has started and the activity was about to
+be correctly performed, just there was some unexpected error during this
+process.
+
+
+### SRT_REJX_UNIMPLEMENTED
+
+The request was correctly recognized, but the current version doesn't
```

Comment:
"current version" of what? SRT? The application? Both? Later you refer to a "server version". How does this relate to SRT and/or the application?

Reply:
It's all about the server version. The rejection codes concern exclusively a situation when you have a service hanging as "SRT Listener" and a client tries to connect, with having a query specified in the StreamID field. Although yes, it should be clarified that it's about the version of the server application (can be both SRT or any other part of the service software).